test/unit/org/apache/cassandra/service/accord/serializers/CommandsForKeySerializerTest.java

```diff
@@ -465,16 +469,46 @@ public void test()
                     next = txnIdGen.next(rs0);
                 return next;
             }).unique().ofSizeBetween(0, 10).next(rs);
+            Arrays.sort(ids, Comparator.naturalOrder());
```

Comment:
moved the sort here so its easier to check "missing".  since we create the TxnInfo in the order of TxnId this doesn't have an impact there.